src/mmw/js/src/modeling/controllers.js

```diff
@@ -35,26 +38,29 @@ var ModelingController = {
                 name: 'Untitled Project',
                 created_at: Date.now(),
                 area_of_interest: App.map.get('areaOfInterest'),
-                scenarios: new models.ScenariosCollection([
-                    new models.ScenarioModel({
-                        name: 'Current Conditions',
-                        is_current_conditions: true,
-                        active: true
-                    })
-                ])
+                scenarios: new models.ScenariosCollection()
+            });
+
+            App.currProject = project;
+
+            var currentConditionsScenario = new models.ScenarioModel({
+                name: 'Current Conditions',
+                is_current_conditions: true,
+                active: true
             });
+            project.get('scenarios').add(currentConditionsScenario);
+
+            initViews(project);
 
             project.on('change:id', function(model) {
                 router.navigate('model/' + model.id);
             });
 
-            var scenariosColl = project.get('scenarios'),
+            var scenarios = project.get('scenarios'),
                 mapView = App.getMapView();
-            scenariosColl.on('change:activeScenario', function(scenario) {
+            scenarios.on('change:activeScenario', function(scenario) {
                 mapView.updateModifications(scenario.get('modifications'));
             });
-
-            initViews(project);
```

Comment:
Was there a reason we had to move this line and change how `ScenariosCollection` gets created here? Or just preference?